modules/exploits/linux/http/empire_skywalker.rb

```diff
@@ -25,11 +32,15 @@ def initialize(info = {})
       'Author'          =>
         [
           'Spencer McIntyre',   # Vulnerability discovery & Metasploit module
-          'Erik Daguerre'       # Metasploit module
+          'Erik Daguerre',      # Metasploit module
+          'ACE-Responder',      # Advance exploit
+          'Takahiro Yokoyama'   # Update Metasploit module
         ],
       'License'         => MSF_LICENSE,
       'References'      => [
-        ['URL', 'http://www.harmj0y.net/blog/empire/empire-fails/']
+        ['URL', 'http://www.harmj0y.net/blog/empire/empire-fails/'],
```

Comment:
It would be good to add a cve ref too: https://nvd.nist.gov/vuln/detail/CVE-2024-6127

Reply:
Thank you for reviewing!
I added cve ref and updated broken link. https://github.com/rapid7/metasploit-framework/pull/19331/commits/61754f3c92cedb85c9b712b75c2f867df14e2878